app/ocr.py

```diff
@@ -189,22 +206,28 @@ def get_date(self) -> str:
         return '{}-{}-{}'.format(sorted_dates[0].year, sorted_dates[0].month, sorted_dates[0].day)
 
     def get_panel(self) -> str:

```

Comment:
Function should output a list of strings.
```suggestion
    def get_panel(self) -> List[str]:
```

Reply:
Changed in updated commit